pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -171,7 +171,7 @@ func createRoute(action *k8s.HTTPRouteAction, ns string) []*istio.HTTPRouteDesti
 		Destination: &istio.Destination{
 			// TODO cluster.local hardcode
 			// TODO is this the right format?
-			Host: fmt.Sprintf("%s.%s.svc.cluster.local", action.ForwardTo.Name, ns),
+			Host: fmt.Sprintf("%s.%s.svc." + constants.DefaultKubernetesDomain, action.ForwardTo.Name, ns),
```

Comment:
using string concat inside of fmt.Sprintf seems weird

Reply:
yeah it's weird ... just trying to avoid any additional overhead.